feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/data/repository/ConvictionVotingRepository.kt

```diff
@@ -45,8 +46,7 @@ interface ConvictionVotingRepository {
 
     fun ExtrinsicBuilder.vote(referendumId: ReferendumId, vote: AccountVote)
 
-    fun ExtrinsicBuilder.vote(votes: Map<ReferendumId, AccountVote>)
-
+    fun CallBuilder.vote(referendumId: ReferendumId, vote: AccountVote)
```

Comment:
Just realized we should add `CallBuilder` to sdk and let `ExtrinsicBuilder` implement it. It would allow us to have a single extension in such places 
Thats just a thought for the next time we modify sdk around ExtrinsicBuilder